lib/Core/Executor.cpp

```diff
@@ -363,6 +363,12 @@ cl::opt<double> MaxStaticCPSolvePct(
              "instructions (default=1.0 (always))"),
     cl::cat(TerminationCat));
 
+cl::opt<std::string> MaxStaticPctCheckDelay(
```

Comment:
I see the point of this option, but can this be disabled by default?
I find non-zero time-based default options a potential cause for non-deterministic behavior.

Reply:
This is just replacing  the ugly hard-coded `statsTracker->elapsed() > time::seconds(60))`.  And note that it is irrelevant in the default configuration because all `MaxStaticPct*` are 1.